src/docs/admin/ProActiveAdminGuide.adoc

```diff
@@ -194,11 +196,11 @@ USER_NAME=activeeon
 GROUP_NAME=activeeon
 # Required to run docker in docker on Linux, look at /etc/group or run 'getent group docker | cut -d: -f3' to get the host docker gid
 DOCKER_GID=115
-# Number of ProActive nodes to start on the server machine
+# Number of ProActive Nodes to start on the server machine
 PROACTIVE_WORKER_NODES=7
-# Proactive admin password
+# ProActive admin password
 PROACTIVE_ADMIN_PASSWORD=activeeon
```

Comment:
ProActive admin password (please change it in the configuration for security reasons)

Reply:
@nebilbenmabrouk Why would it be a security threat to have activeeon as a password example? See here: https://github.com/ow2-proactive/documentation/pull/742/files#diff-429ab7d50b78abcfc341af9b172b38bfd1b73e6fa3907ae439215346a59caf0cR495

Reply:
@mboussaa because the password is written in publicly accessible documentation, and system admins should change it

Reply:
@nebilbenmabrouk here it is like saying u can use locally admin/admin which is also public in our doc https://trydev2.activeeon.com/doc/admin/ProActiveAdminGuide.html#_file